salt/modules/gentoo_service.py

```diff
@@ -37,15 +97,8 @@ def get_enabled():
 
         salt '*' service.get_enabled
     '''
-    ret = set()
-    lines = __salt__['cmd.run']('rc-update show').splitlines()
-    for line in lines:
-        if '|' not in line:
-            continue
-        if 'shutdown' in line:
-            continue
-        ret.add(line.split('|')[0].strip())
-    return sorted(ret)
+    (enabled_services, disabled_services) = _get_service_list()
+    return collections.OrderedDict(enabled_services)
```

Comment:
This won't work on Python 2.6, which Salt needs to support for the time being. Please have a look at `salt.utils.odict` and migrate to that if it will work for you.